### PR TITLE
Standalone search announcement

### PR DESCRIPTION
**Pull request by leggetter**

### CODE REVIEW

File: blog/2023-05-25-tigris-standalone-search/index.mdx [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+};
+
+Database search is okay, at best. To build awesome search experiences you need a
+dedicated search product.
+
+Earlier this month we
+[released Tigris Search in BETA](/blog/tigris-database-search-sync-beta/) and
+covered how it supports automatic syncronization between Tigris Database and
+Tigris Search, which improves developer productivity through enabling the team
+to focus on shipping features and not setting up and managing database-to-search
```
Comment (ovaistariq) [inline blog/2023-05-25-tigris-standalone-search/index.mdx:29]:
```suggestion
to focus on shipping features and not setting up and managing the search and 
```

File: blog/2023-05-25-tigris-standalone-search/index.mdx [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+syncronization infrastructure.
+
+But, you many not yet be in the position to migrate all your data over to Tigris
```
Comment (ovaistariq) [inline blog/2023-05-25-tigris-standalone-search/index.mdx:32]: However, you may be looking to quickly enhance your application experience by adding the capability to search existing data without waiting to migrate your database. Or, your data may not be in a database; maybe you've got all your data in S3.

File: blog/2023-05-25-tigris-standalone-search/index.mdx [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+and you may have a more pressing need to enhance your product with a great
+search experience. Or, you may not store your data in a database; maybe you've
+got all your data in S3, or you've got a statically generated site driven by a
+number of data files stored in your source control.
+
+Whatever the reason, Tigris Standalone Search can help if you're looking for a
```
Comment (ovaistariq) [inline blog/2023-05-25-tigris-standalone-search/index.mdx:38]: Tigris Standalone Search feature allows you to quickly add search to your existing data.

Comment (ovaistariq) [reply blog/2023-05-25-tigris-standalone-search/index.mdx:38]: The important point is that it enables developers to quickly add search. This is essentially an iterative way to adopt Tigris, by moving to Tigris workload at a time.

File: blog/2023-05-25-tigris-standalone-search/index.mdx [hunk 4]
```diff
@@ -0,0 +1,528 @@
...
+search solution.
+
+[![Enabling awesome search experiences with Tigris Standalone Search](./tigris-standalone-search.png)](/blog/tigris-standalone-search/)
+
+<!-- truncate -->
+
+## Tigris Standalone Search is available with Tigris Search BETA
+
+As part of our release of Tigris Search BETA we also made it possible for your
```
Comment (ovaistariq) [inline blog/2023-05-25-tigris-standalone-search/index.mdx:47]:
```suggestion
As part of our release of Tigris Search BETA we also made it possible for you
```

File: blog/2023-05-25-tigris-standalone-search/index.mdx (hunk 3 above)
Comment (ovaistariq) [reply blog/2023-05-25-tigris-standalone-search/index.mdx:38]: Let's not mention `got a statically generated site driven`. As that is not the use case we are targeting.

**Review by himank — APPROVED** [review]

File: blog/2023-05-25-tigris-standalone-search/index.mdx [hunk 5]
```diff
@@ -0,0 +1,629 @@
+---
+slug: tigris-standalone-search
+title: Enabling awesome search experiences with Tigris Standalone Search
+description: >
+  Enabling awesome search experiences with Tigris Standalone Search
+keywords: [nosql search, standalone search, search engine, search experiences]
+authors: [leggetter]
+tags: [nosql search, standalone search]
```
Comment (ovaistariq) [inline blog/2023-05-25-tigris-standalone-search/index.mdx:8]: Another tag would be good: `full-text search`

**Review by ovaistariq — APPROVED** [review]

**Review by rbarabas — APPROVED** [review]
Looks GTM! 

Btw the PR is still in Draft, lmk if you need a review once it's ready to go!

**Pull request merged**